local-cli/link/windows/patches/makePackagePatch.js

```diff
@@ -0,0 +1,10 @@
+const applyParams = require('./applyParams');
+
+module.exports = function makePackagePatch(packageInstance, params, prefix) {
+  const processedInstance = applyParams(packageInstance, params, prefix);
+
+  return {
+    pattern: 'new MainReactPackage()',
```

Comment:
Unless you use the `flip` parameter here, then you can add the comma after `processedInstance` to guarantee it's always correct.

---
In reply to: [90885370](https://github.com/facebook/react-native/pull/11282#discussion_r90885370) [](ancestors = 90885370)